macos/library/FLEFileChooserPlugin.m

```diff
@@ -30,11 +30,15 @@
 static NSString *const kPlatformMethodNameKey = @"method";
 static NSString *const kPlatformMethodArgsKey = @"args";
 
-// Method argument keys.
+// Method argument keys for NSSavePanel.
+static NSString *const kOKButtonLabelKey = @"prompt";
```

Comment:
My concern about this line being overly platform-specific was about the string value, which is the part that is a cross-platform API, not the variable name. The Dart and other desktop implementations should not have to call this argument "prompt" (which doesn't communicate much information about what it is) just because that's what the NSSavePanel calls the property.

That's the part I think should be called confirmButtonLabel or similar ("ok" is, again, macOS-specific)

Reply:
That makes sense to me, thanks.